### ISSUE TITLE
Two warnings to fix before a clean build is achieved on clang 11

### ISSUE THREAD

**Issue by deuteragenie** [performous/performous#618]
```
[36/71] Compiling C++ object performous.p/game_pitch.cc.o00:43
../game/pitch.cc:58:11: warning: variable length arrays are a C99 feature [-Wvla-extension]00:43
 00:43
                 ^00:43
1 warning generated.00:43
[46/71] Compiling C++ object performous.p/game_screen_intro.cc.o00:58
In file included from ../game/screen_intro.cc:1:00:58
../game/screen_intro.hh:35:7: warning: private field 'm_drawNotice' is not used [-Wunused-private-field]00:58
        bool m_drawNotice = false;
```

**Comment by Tronic** [comment]
Easy to fix. Replace the VLA with std::vector the same way the infoLog was replaced, and remove the unused field.

**Comment by twollgam** [comment]
I will fix it and create a pr.

**Comment by Baklap4** [comment]
I think it's already fixed in one pr https://github.com/performous/performous/pull/620/files

**Comment by twollgam** [comment]
OK, thanks for letting me know.

BTW I think the fix in pr #620 is not correct and has to look like this in screen_intro.hh:

```
@@ -32,7 +32,9 @@ class ScreenIntro : public Screen {
 	std::unique_ptr<ThemeIntro> theme;
 	Menu m_menu;
 	bool m_first;
+#ifdef USE_WEBSERVER
 	bool m_drawNotice = false;
+#endif
 	AnimValue m_selAnim;
 	AnimValue m_submenuAnim;
 	AnimValue m_webserverNoticeTimeout;

```

**Comment by Baklap4** [comment]
@twollgam please add it as comment to the PR then it can be fixed there 👍

**Comment by Baklap4** [comment]
closed through #620
